lib/bootloader_setup.pm

```diff
@@ -35,17 +36,32 @@ our @EXPORT = qw(
   tianocore_select_bootloader
 );
 
+# prevent grub2 timeout; 'esc' would be cleaner, but grub2-efi falls to the menu then
+sub stop_grub_timeout {
+    send_key 'up';
+}
+
 sub boot_local_disk {
     if (check_var('ARCH', 'ppc64le')) {
         # TODO use bootindex to properly boot from disk when first in boot order is cd-rom
         wait_screen_change { send_key 'ret' };
-        if (check_screen 'inst-slof', 5) {
-            # specify local disk for boot
+        assert_screen [qw(inst-slof bootloader grub2 inst-bootmenu)];
```

Comment:
Congratulation, you finally managed to get rid of you enemy check_screen 😆 I don't mind, it's just duplicate.
All this tags are already asserted before boot_local_disk